client/components/common/ActivityBrowser/SelectActivity.vue

```diff
@@ -32,7 +32,7 @@
 
 <script>
 import activityApi from 'client/api/activity';
-import CircularProgress from 'components/common/CircularProgress';
+import { CircularProgress } from 'tce-core';
```

Comment:
Can we remove it entirely and just use vuetify one which is registrated globally ❓